mypy/util.py

```diff
@@ -517,6 +517,8 @@ def hash_digest(data: bytes) -> str:
 
 def parse_gray_color(cup: bytes) -> str:
     """Reproduce a gray color in ANSI escape sequence"""
+    if sys.platform == "win32":
+        assert False, "curses is not available on Windows"
```

Comment:
```suggestion
    assert sys.platform != "win32", "curses is not available on Windows"
```
This should work, right?

Reply:
I tried that first, doesn't seem to work unfortunately

Reply:
Ironically the only reason why `curses` is causing the self-check to fail is because of my own PR to typeshed https://github.com/python/typeshed/pull/6749